pkg/model/workflow.go

```diff
@@ -670,3 +664,17 @@ func (w *Workflow) GetJobIDs() []string {
 	}
 	return ids
 }
+
+var OnDecodeNodeError = func(node yaml.Node, out interface{}, err error) {
```

Comment:
So it's possible to ignore errors when decoding nodes by setting `OnDecodeNodeError` to nil.